fact_sheets_category_1.yml

```diff
@@ -0,0 +1,392 @@
+schema_version: '0.3'
+os_version: 3.5.1
+os_sha: 22e1db7be5
+buildstock_directory: ../ # Relative to this file or absolute
+project_directory: project_national # Relative to buildstock_directory
+output_directory: national_baseline
+# weather_files_url: https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip
+weather_files_path: /shared-projects/buildstock/weather/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: residential_quota
+  args:
+    n_datapoints: 2200000
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: hourly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+# Federal Minimum Category
+upgrades:
+  - upgrade_name: Federal Minimum Heating
```

Comment:
What about 
```
Fuel Oil Fuel Wall/Floor Furnace
Natural Gas Fuel Wall/Floor Furnace
Propane equivalent?
Electric furnace, boiler, baseboard, wall furnace?
```?

Reply:
My impression is that there aren't federal minimums for these technologies.
1. We should verify this with Jeff Munk
2. We should probably include a "like replaces like" so that we are costing them appropriately. e.g. the NG wall/floor furnace is replaced by an identical NG wall/floor furnace, but there's a cost associated with that